-------------------------------------------------------------------------------
Test set: jsonIO.AerolineaTests
-------------------------------------------------------------------------------
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.288 s <<< FAILURE! -- in jsonIO.AerolineaTests
jsonIO.AerolineaTests.testValidacionesDeCampos -- Time elapsed: 0.054 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <jakarta.validation.ConstraintViolationException> but was: <java.lang.IllegalArgumentException>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3111)
	at jsonIO.AerolineaTests.testValidacionesDeCampos(AerolineaTests.java:105)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.IllegalArgumentException: El nombre no puede estar vac√≠o
	at controladores.AerolineaControlador.crear(AerolineaControlador.java:31)
	at jsonIO.AerolineaTests.lambda$testValidacionesDeCampos$5(AerolineaTests.java:106)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)
	... 6 more

